README.adoc

```diff
@@ -307,9 +307,9 @@ the `localhost` host to the inventory. Next, the test case makes a connection to
 Then, it asserts whether the time that is needed to retrieve
 the system properties for localhost is less than 4 seconds.
 
-* The [hotspot=testInventoryAccessCountMetric file=0]`testInventoryAccessCountMetric()` test case validates the [hotspot=countedForList file=1]`@Counted` metric. The test case sends a request to the
-`\http://localhost:9080/inventory/systems` URL to retrieve the whole inventory, and then it asserts
-that this endpoint is accessed at least once.
+* The [hotspot=testInventoryAccessCountMetric file=0]`testInventoryAccessCountMetric()` test case validates the [hotspot=countedForList file=1]`@Counted` metric. 
+The test case obtains the metric before and after a request to the `\http://localhost:9080/inventory/systems` URL.
+It asserts that the metric was increased after the access.
```

Comment:
For this one, I'm thinking update it to 

>It then asserts that the metric was increased after the URL was accessed.

If that makes sense to you? I had a harder time understanding exactly what this sentence meant.